src/components/Lpcontent/Lpcontent.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import './Lpcontent.css';
```

Comment:
CSS file should be imported to the same file as the class, in this case the file should be imported to Topservicescard.js